blutter/src/DartDumper.cpp

```diff
@@ -82,6 +82,110 @@ static std::string getFunctionName4Ida(const DartFunction& dartFn, const std::st
 	return prefix + fnName;
 }
 
+static bool is_valid_char(const char ch) {
+	if ((ch >= 'a' && ch <= 'z') || (ch >= 'A' && ch <= 'Z') || isdigit(ch)) {
+		return true;
+	}
+	switch (ch) {
+	case '.':
+	case ':':
+	case '_':
+		return true;
+	default:
+		return false;
+	}
+}
+
+static void filterString(std::string &str) {
+	for (char& ch : str) {
+		if (!is_valid_char(ch)) {
+			ch = '_';
+		}
+	}
+}
+
+void DartDumper::Dump4Radare2(std::filesystem::path outDir)
+{
+	std::filesystem::create_directory(outDir);
+	std::ofstream of((outDir / "addNames.r2").string());
+	of << "# create flags for libraries, classes and methods\n";
+
+	of << std::format("f app.base = {:#x}\n", app.base());
+	of << std::format("f app.heap_base = {:#x}\n", app.heap_base());
```

Comment:
I think these are un-necessary since they change with every run and aren't constant.

Reply:
but its good to have some as reference, but yeah i was trying to expose some pointers for later testing and see if those are useful.

Reply:
what would be nice is to know where the object pool is located. aka the address of PP. because right now r2 bases it to address 0, but its configurable. and blutter can be specifying that address